test/format/shortfnsyntax.d

```diff
@@ -0,0 +1,8 @@
+int bingBong(int x) => x + 1;
+
+int ________________________________________________________(
+	int x) => x + 1;
```

Comment:
Can you add test with a shorter function name so we can be confident things preferably break before the `=>` (they do, right?).

Reply:
```
int __________________________(x) => x
        + 1;
```
is a fixed point currently.

Is there notion for "break here if you need to, it's cheap" without a span at the moment?

Reply:
hang on forgot the type for x

Reply:
That's what span are for.

Reply:
we're spannin' now.